CONTRIBUTING.md

```diff
@@ -62,3 +63,47 @@ variable:
 ```bash
 $ export EXERCISES_API_URL=http://localhost:9292
 ```
+
+### Track Anatomy
+
+Each track should have the following structure:
+
+```bash
+├── .gitignore
+├── .travis.yml
+├── LICENSE
+├── README.md
+├── SETUP.md
+├── bin
+│   └── fetch-configlet
+├── config.json
+├── docs
+│   ├── ABOUT.md
+│   ├── INSTALLATION.md
+│   ├── LEARNING.md
+│   ├── RESOURCES.md
+│   └── TESTS.md
+└── exercises
+    └── hello_world
+        ├── hello_world_example.file
+        ├── hello_world.file
+        └── hello_world_test.file
+```
+
+* `LICENSE` - The MIT License (MIT)
+* `README.md` - a thorough explanation of how to contribute to the track.
+* `SETUP.md` - ???
+* `bin` - scripts and other files related to running the track's tests, etc.
+* `config.json` - ???
+* `docs` - the documentation for the track.These files are served to the exercism.io help site via the x-api. It should contain at minimum:
+
+    - `INSTALLATION.md` - about how to get the track's language set up locally.
+    - `TESTS.md` - how to run the tests for the track's individual exercises.
+
+    Some nice to haves:
+
+    - `ABOUT.md` - a short, friendly blurb about the track's language. What types of problems does it solve really well? What is it typically used for?
+    - `LEARNING.md` - a few notes about where people might want to go to learn the track's language from scratch.
+    - `RESOURCES.md` - references and other useful resources.
```

Comment:
Difference between learning and resources: learning is for first getting up to speed in a language, resources would be for continuous development (e.g. standard library docs, useful tools, etc).